tests/components/comelit/test_config_flow.py

```diff
@@ -47,6 +47,38 @@ async def test_user(hass: HomeAssistant) -> None:
     assert mock_setup_entry.called
 
 
+async def test_user_vedo(hass: HomeAssistant) -> None:
+    """Test starting a flow by user."""
+    with patch(
+        "aiocomelit.api.ComelitVedoApi.login",
+    ), patch(
+        "aiocomelit.api.ComelitVedoApi.logout",
+    ), patch(
+        "homeassistant.components.comelit.async_setup_entry"
```

Comment:
You could parametrize your old bridge tests.

```suggestion
@pytest.mark.parametrize(
    ("class", "user_input"),
    [
        ("ComeliteSerialBridgeApi", MOCK_USER_BRIDGE_DATA),
        ("ComelitVedoApi", MOCK_USER_VEDO_DATA),
    ]
)
async def test_full_flow(hass: HomeAssistant, class: str, user_input: dict[str, Any]) -> None:
    """Test starting a flow by user."""
    with patch(
        f"aiocomelit.api.{class}.login",
    ), patch(
        f"aiocomelit.api.{class}.logout",
    ), patch(
        "homeassistant.components.comelit.async_setup_entry"
...
```

Reply:
Thx for the suggestion! Applied